### PR TITLE
xtest: stats: used exported PTA API + print clock/regulator trees

### PR DESCRIPTION
**Pull request by etienne-lms**
This P-R depends on https://github.com/OP-TEE/optee_os/pull/6521.

### CODE REVIEW

File: host/xtest/stats.c [hunk 1]
```diff
@@ -59,6 +33,8 @@ static int usage(void)
 	fprintf(stderr, " --memleak      Dump memory leak data on secure console\n");
 	fprintf(stderr, " --ta           Print loaded TAs context\n");
 	fprintf(stderr, " --time         Print REE and TEE time\n");
+	fprintf(stderr, " --clocks       Dump clock tree on secure console\n");
+	fprintf(stderr, " --regulators   Dump regulator tree on secure console\n");
```
Comment (jforissier) [inline host/xtest/stats.c:37]: Wrong commit

**Comment by etienne-lms** [comment]
[comment above](https://github.com/OP-TEE/optee_test/pull/717#discussion_r1414021856) addressed.

**Review by jforissier — APPROVED** [review]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by etienne-lms** [comment]
review tag applied

**Pull request merged**